index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Quiz app</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <form id="form">
+      <div id="question">
+        <label for="question">Enter your question</label>
+        <input id="question-input" type="text" name="question" />
+      </div>
+      <div id="answers">
+        <div class="answer-option">
+          <label for="option1"></label>
+          <input id="option1" type="text" name="option1" />
```

Comment:
a lot of repetitions here but this week we will be learning how to make this structure in couple lines of code instead! :)

Reply:
Yes, I have removed the multiple usage of getElementById and access from its parent variable Form. Thank you for the review.